frame/lottery/src/tests.rs

```diff
@@ -30,7 +30,7 @@ fn initial_state() {
 	new_test_ext().execute_with(|| {
 		assert_eq!(Balances::free_balance(Lottery::account_id()), 0);
 		assert!(crate::Lottery::<Test>::get().is_none());
-		assert_eq!(Participants::<Test>::get(&1), (0, vec![]));
```

Comment:
please add a couple of simple tests that things behave as expected with this new limit.

Reply:
For example a simple test for `calls_to_indices` and `participating_calls.try_push` logic.

Reply:
The limit already existed before, the only function where it could happen is `set_calls` which already has a test.  
Speaking of which: `stop_repeat` does not have any testing :thinking:

Reply:
Added a test for `calls_to_indices`, but for the `participating_calls.try_push` case see my comment in the discussion above.

Reply:
happy for you to also add a quick test for `stop_repeat` :)

Reply:
So I added a white-box test for the `do_buy_ticket` function, I think this is as close as I can get to testing the `participating_calls.try_push`.